v2/concordium/types.proto

```diff
@@ -2132,3 +2132,30 @@ message Branch {
   // Further blocks branching of this block.
   repeated Branch children = 2;
 }
+
+// The leadership election nonce is an unpredictable value updated once an
+// epoch to make sure that bakers cannot predict too far in the future when
+// they will win blocks.
+message LeadershipElectionNonce {
+  bytes value = 1;
+}
+
+// Response type for GetElectionInfo.
+// Contains information related to baker election for a perticular block.
+message ElectionInfo {
+  message Baker {
+    // The ID of the baker.
+    BakerId baker = 1;
+    // The account address of the baker.
+    AccountAddress account = 2;
+    // The approximate lottery power of the baker.
```

Comment:
Please explain what "approximate" means. Something like "Rounded to the nearest representable "double"".